.github/workflows/mirror-dev.yml

```diff
@@ -0,0 +1,34 @@
+name: Mirror dev to GitLab
+
+on:
+  push:
+    branches:
+      - dev
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - dev
```

Comment:
Could you please provide some info on why we'd need to run this when a PR is submitted?

Reply:
Looked further into PR trigger and removed.